plugins/irisv12/go.mod

```diff
@@ -0,0 +1,39 @@
+module github.com/apache/skywalking-go/plugins/irisv12
+
+go 1.18
+
+require (
+	github.com/apache/skywalking-go/plugins/core v0.0.0-20230825023739-685f598ab343
```

Comment:
Please delete this package declaration since skywalking-go uses workspace to manage all packages.